src/app/components/isAnswerCorrect.js

```diff
@@ -0,0 +1,11 @@
+const isAnswerCorrect = (guess, answer) =>{
+    if(typeof guess !== "string"){
+        throw new Error("TypeError");
```

Comment:
```suggestion
        throw new TypeError("Guessed answer isn't string");
```

See docs for more info https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError